providers.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.0"
+      version = ">= 4.0"
```

Comment:
```suggestion
      version = ">= 4.24.0"
```

Reply:
Updated

Reply:
Except it's not updated?

Reply:
I removed providers.tf.  You had me add a versions.tf file.  I missed that it caused a duplicate required_providers block.